### PR TITLE
Send Lambda logs to Papertrail

### PR DESCRIPTION
**Pull request by the-smooth-operator**

### CODE REVIEW

**Comment by the-smooth-operator** [comment]
All Lambda logs are now going to Papertrail, enjoy!

File: terraform/cloudwatch-logs-to-papertrail.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  runtime       = "nodejs10.x"
+  role          = "${aws_iam_role.logs_to_papertrail.arn}"
+}
+
+resource "aws_lambda_permission" "allow_cloudwatch_to_lambda_logs" {
+  statement_id  = "AllowExecutionFromCloudWatch"
+  action        = "lambda:InvokeFunction"
+  function_name = "${aws_lambda_function.logs_to_papertrail.function_name}"
+
+  #principal     = "events.amazonaws.com"
```
Comment (LeoMcA) [inline terraform/cloudwatch-logs-to-papertrail.tf]: Is this comment needed?

File: terraform/cloudwatch-logs-to-papertrail.tf [hunk 2]
```diff
@@ -0,0 +1,48 @@
+resource "aws_s3_bucket" "logs_to_papertrail" {
+  bucket = "logs-to-papertrail-lambda-code"
```
Comment (LeoMcA) [inline terraform/cloudwatch-logs-to-papertrail.tf:2]: Do we have the source code for the contents of this bucket listed somewhere?

**Review by LeoMcA — APPROVED** [review]

File: terraform/cloudwatch-logs-to-papertrail.tf (hunk 1 above)
Comment (the-smooth-operator) [reply terraform/cloudwatch-logs-to-papertrail.tf]: it is not

File: terraform/cloudwatch-logs-to-papertrail.tf (hunk 2 above)
Comment (the-smooth-operator) [reply terraform/cloudwatch-logs-to-papertrail.tf:2]: good point. I'll add a comment there pointing to the source repo

Comment (LeoMcA) [reply terraform/cloudwatch-logs-to-papertrail.tf:2]: we might want to loop back and do the same with the tldr and email-in lambdas as well

**Comment by the-smooth-operator** [comment]
thanks for the review @LeoMcA !

File: terraform/cloudwatch-logs-to-papertrail.tf (hunk 2 above)
Comment (the-smooth-operator) [reply terraform/cloudwatch-logs-to-papertrail.tf:2]: yes, just send me the links to the repos. Or feel free to PR!

**Pull request merged**